lib/internal/util.js

```diff
@@ -560,6 +562,14 @@ function SideEffectFreeRegExpPrototypeExec(regex, string) {
   return FunctionPrototypeCall(RegExpFromAnotherRealm.prototype.exec, regex, string);
 }
 
+function isArrayBufferDetached(value) {
+  if (ArrayBufferPrototypeGetByteLength(value) === 0) {
+    return _isArrayBufferDetached(value);
+  }
+
+  return false;
```

Comment:
Nit: the flow might look semantically better like this
```suggestion
  if (ArrayBufferPrototypeGetByteLength(value) !== 0) {
    return false;
  }

  return _isArrayBufferDetached(value);
```